suite-native/module-graph/src/dummyData.ts

```diff
@@ -0,0 +1,18 @@
+import { GraphPoint } from 'react-native-graph';
+
+// @ts-expect-error
+import gaussian from 'gaussian';
```

Comment:
I am not sure about this lib, why do we need it?

Reply:
We don't need it absolutely :). I just copied this from [official example](https://github.com/margelo/react-native-graph/blob/main/example/src/data/GraphData.ts) of the [library](https://github.com/margelo/react-native-graph/tree/main/example) because it looked cool to me :D. I have removed it and replaced it with good old value: Math.floor and Math.random() combination :).